packages/api/src/zodSchema/authSchema.ts

```diff
@@ -0,0 +1,6 @@
+import { z } from "zod";
+
+export const authSchema = z.object({
+  password: z.string().min(8),
```

Comment:
I would probably avoid this, since if you change your rules overtime you might end up locking people out of their accounts.

```suggestion
  password: z.string(),
```